plugins/zulip/alerta_zulip.py

```diff
@@ -1,49 +1,39 @@
 import logging
 import os
+from datetime import datetime
+
+import zulip
+from jinja2 import Template, UndefinedError
+
+from plugins.zulip.config.static_config import DATABASE, DB_ROWS
+from plugins.zulip.database import DBHelper
 
 try:
     from alerta.plugins import app  # alerta >= 5.0
 except ImportError:
     from alerta.app import app  # alerta < 5.0
 from alerta.plugins import PluginBase
 
-import zulip
-from jinja2 import Template, UndefinedError
-
+LOG = logging.getLogger('alerta.plugins.zulip')
 
-DEFAULT_TMPL = """
-{% if customer %}Customer: `{{customer}}` {% endif %}
-*[{{ status.capitalize() }}] {{ environment }} {{ severity.capitalize() }}*
-{{ event }} {{ resource.capitalize() }}
-```
-{{ text }}
-```
-"""
+ZULIP_API_KEY = app.config.get('ZULIP_API_KEY') or os.environ.get('ZULIP_API_KEY')
+ZULIP_EMAIL = app.config.get('ZULIP_EMAIL') or os.environ.get('ZULIP_EMAIL')
+ZULIP_SITE = app.config.get('ZULIP_SITE') or os.environ.get('ZULIP_SITE')
+ZULIP_TYPE = app.config.get('ZULIP_TYPE') or os.environ.get('ZULIP_TYPE', 'stream')
+ZULIP_TO = app.config.get('ZULIP_TO') or os.environ.get('ZULIP_TO')
+ZULIP_SUBJECT = app.config.get('ZULIP_SUBJECT') or os.environ.get('ZULIP_SUBJECT', 'APImon alert')
+ZULIP_ALLOW_UNSECURE = app.config.get('ZULIP_ALLOW_UNSECURE') or os.environ.get('ZULIP_ALLOW_UNSECURE')
+DB_CONNECTION_STRING = app.config.get('DB_CONNECTION_STRING') or os.environ.get('DB_CONNECTION_STRING')
```

Comment:
DATABASE_URL currently already holds DB connection, let's reuse it (at least as a fallback)